### ISSUE TITLE
New ShowUI breaks my User Interface

### ISSUE THREAD

**Issue by fschottmann** [ShowUI/ShowUI#4]
If i switch to the new update (2.0 ; previous was 1.5) it breaks my whole user interface there are many major errors in the new update, e.g. when i set New-DataGrid -CanUserReorderColumns $true -CanUserResizeColumns $true -CanUserSortColumns $true it gives me an error like this [33564,9: Set-WpfProperty] Cannot find an overload for "Add" and the argument count: "1". Any plans to resolve this issue?

**Comment by Jaykul** [comment]
Yes -- you're using the `dev` branch for a major release, so _some_ breakage is expected ;-)

### But I definitely don't want to break everything.  

If you can help me with some additional examples (i.e. give me a version of your script so I can make sure it works)  ... or even help by writing some actual test cases (don't ask me why we've never had any) ... it would be greatly appreciated.

**Comment by johnmbaughman** [comment]
https://github.com/ShowUI/ShowUI/blob/dev/Examples/firstnamelastname.ps1
Your own example shows it.

Using this version of PowerShell and the latest dev instance of ShowUI.

`Major  Minor  Build  Revision`
`-----  -----  -----  --------`
`5        1      16299  785`

If I get some time soon, I'll take a look and see what I find. 

(I'm just looking into WPF and PowerShell just for the fun of it, but I'd like to help out if I can.)
